server/events/policy_check_output_generator.go

```diff
@@ -0,0 +1,113 @@
+package events
```

Comment:
can we stop overloading this package? events is large and completely unmaintainable/readable.

Reply:
Sure. I originally had it outside of events package but since `PolicyCheckCommandOutputGenerator` has dependencies on `events.ProjectPolicyCheckCommandRunner` and `events.ProjectPlanCommandBuilder` and `ApprovePoliciesCommandRunner` has a dependency on `PolicyCheckOutputGenerator`, from my understanding there are two possible approaches:
- Define the `CommandOutputGenerator` interface in the events package so that `ApprovePoliciesCommandRunner` does not have any dependency on the `PolicyCheckOutputGenerator`.  When going this route, I realized the the interface method returns a `PolicyCheckOutputStore` object which is populated by the `PolicyCheckOutputGenerator`. We could move this object into the models package and have exported getters and setters to break the dependency cycle.

- Move ApprovePoliciesCommandRunner and PolicyCheckOutputGenerator in the same package (not in events) maybe `server/events/approve_policies`. This could be possible but it could get confusing if some command runners are in the events package while one is not. 

Maybe I'm missing an easier route 😅, do you have any suggestions?

Reply:
an example of how we could do this is by using the command package https://github.com/lyft/atlantis/blob/9dde8b440fd790e62e40ffb9defe79706b37a9ee/server/events/command/comment.go#L57

use your best judgement here and think about the long term and how we can move towards that incrementally.

Reply:
Moved `policy_check_output_generator` in to `events/commands/policies`, `PolicyCheckOuputStore` into `events/commands` and defined the `CommandOutputGenerator` interface in `approvePoliciesCommandRunner`.